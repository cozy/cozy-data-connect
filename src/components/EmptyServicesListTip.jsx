import React from 'react'

import { Title, Text } from 'cozy-ui/react/Text'
import { Media, Img, Bd } from 'cozy-ui/transpiled/react/Media'
import { translate } from 'cozy-ui/react/I18n'
import ArrowIllustration from 'assets/images/drawing-arrow-up.svg'

export const EmptyServicesListTip = ({ t }) => (
  <Media align="top" className="EmptyServicesListTip">
    <Img>
      <img src={ArrowIllustration} />
    </Img>
    <Bd className="EmptyServicesListTip-text">
      <Title>{t('connector.empty.title')}</Title>
      <Text tag="p" className="u-mv-half">
        {t('connector.empty.text')}
      </Text>
    </Bd>
  </Media>
)

export default translate()(EmptyServicesListTip)
