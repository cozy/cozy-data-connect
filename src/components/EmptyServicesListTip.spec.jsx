import React from 'react'
import { shallow } from 'enzyme'
import { EmptyServicesListTip } from './EmptyServicesListTip'
import { tMock } from '../../test/jestLib/I18n'

describe('EmptyServicesListTip', () => {
  it('should math the snapshot', () => {
    const component = shallow(<EmptyServicesListTip t={tMock} />)
    expect(component.getElement()).toMatchSnapshot()
  })
})
