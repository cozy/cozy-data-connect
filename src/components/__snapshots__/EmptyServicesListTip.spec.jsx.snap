// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyServicesListTip should math the snapshot 1`] = `
<Media
  align="top"
  className="EmptyServicesListTip"
>
  <Img>
    <img
      src="test-file-stub"
    />
  </Img>
  <Bd
    className="EmptyServicesListTip-text"
  >
    <ForwardRef
      ellipsis={false}
      tag="div"
    >
      Start gathering your data!
    </ForwardRef>
    <ForwardRef
      className="u-mv-half"
      ellipsis={false}
      tag="p"
    >
      Synchronise your brands with your Cozy to automatically retrieve your data (bills, reimbursements, expensesâ€¦)
    </ForwardRef>
  </Bd>
</Media>
`;
